curriculum/challenges/english/25-front-end-development/lecture-working-with-html-tools/672a4fa7d335bc7cfb63a392.md

```diff
@@ -10,6 +10,58 @@ dashedName: what-is-an-html-validator
 
 Watch the video lecture and answer the questions below.
 
+# --transcript--
+
+What is an HTML validator, and how can it help you debug your code?
+
+HTML is a very forgiving language – elements still render even when you make mistakes, like forgetting to include a closing tag.
+
+Let's say you have an `h2` element without a closing tag:
+
+```html
+<h1>Article Topic</h1>
+<h2>Subheading 1 </h2>
+<h2>Subheading 2 </h2>
+
+<!-- This h2 does not have a closing tag -->
+<h2>Subheading 3
+```
+
+The `h2` without a closing tag will still render fine. This happens because browsers use a parsing algorithm that handles common errors and tries to render HTML as closely as possible to the author's intention.
+
+But this could backfire sometimes. Let's add a few paragraphs under the existing heading 2 tags in the code:
+
+```html
+<h1>Article Topic</h1>
+<p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Maiores, nisi.</p>
+
+<h2>Subheading 1 </h2>
+<p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. At, doloremque.</p>
+
+<h2>Subheading 2 </h2>
+<p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Unde, placeat.</p>
+
+<!-- This h2 does not have a closing tag -->
+<h2>Subheading 3
+<p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Tempore, illum.</p>
+```
+
+As a result, the paragraph element under the `h2` without a closing `h2` tag renders as heading 2. This is why you need an HTML validator.
+
+An HTML validator is a tool that checks the validity of your HTML code against the standard HTML specifications. It helps you identify errors and warnings in your HTML code, ensuring your web pages are correctly structured and compliant with web standards.
+
+Using an HTML validator benefits not only you and your future code reviews, but also anybody else going through your code, such as your teammates and open-source contributors.
+
+There are several HTML validators out there you can use. The most widely accepted one is the [`w3.org`](http://w3.org) markup validation service.
```

Comment:
```suggestion
There are several HTML validators out there you can use. The most widely accepted one is the `w3.org` markup validation service.
```
Not sure if adding the link is necessary, since there's a link to the validator in the next sentence.